src/tools/ResultsComparer/ResultsComparer.sln

```diff
@@ -0,0 +1,16 @@
+﻿
```

Comment:
Do we need this file?

Reply:
VS is generating it whenever I open the project so I added it just so it stops doing this